flashinfer/jit/gdn.py

```
@@ -24,25 +24,37 @@
     gen_jit_spec,
     sm90a_nvcc_flags,
+    sm120a_nvcc_flags,
 )
 from .utils import write_if_different
 
 
-def gen_gdn_prefill_sm90_module() -> JitSpec:
+def _gen_gdn_prefill_module(arch: str) -> JitSpec:
     """Generate JIT module for GDN prefill kernel with separate compilation.
 
     This generates 32 separate kernel instantiation files (2 dtypes × 16 boolean combinations)
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)